packages/editor/src/components/preferences-modal/index.js

```diff
@@ -152,6 +154,18 @@ export default function EditorPreferencesModal( { extraSections = {} } ) {
 											'Enable pre-publish checks'
 										) }
 									/>
+									{ starterPatterns?.length && (
```

Comment:
This is a little edge-casey, but can we update this to either be a `!!` or a ternary? I noticed while testing in Playground that if the starter patterns haven't loaded yet, this will output a `0` in the UI. You do have to be very quick to open the Preferences panel in order to see this, though (or throttle your network pretty aggressively):

https://github.com/user-attachments/assets/fef7234b-32c0-4087-97c9-58dbae79007c

Also, I noticed that this option will only be available in the preferences if we're on a larger viewport. Is that intentional?

Unrelated to this PR, but I noticed while testing the preferences modal in narrower viewports that the back button within the modal doesn't appear to work. I've logged that in https://github.com/WordPress/gutenberg/issues/65109

Reply:
> Also, I noticed that this option will only be available in the preferences if we're on a larger viewport. Is that intentional?

That's because I bunged it in the publishing section without much thought. I'll move back to Appearance 👍🏻 

Also 👍🏻  on the bool coercion - I always forget that.